opba-protocols/protocol-bpmn-shared/src/main/java/de/adorsys/opba/protocol/bpmnshared/service/context/ContextUtil.java

```diff
@@ -54,7 +59,27 @@
         StandardEvaluationContext parseContext = new StandardEvaluationContext(new SpelCtx<>(execution, context));
         return parser.parseExpression(expression, new TemplateParserContext()).getValue(parseContext, resultClass);
     }
+    public URI buildAndExpandQueryParameters(String urlTemplate, BaseContext context,
+                                             String redirectCode, String scaType) {
+        Map<String, String> expansionContext = new HashMap<>();
 
+        expansionContext.put("sessionId", context.getAuthorizationSessionIdIfOpened());
+        expansionContext.put("wrong", null == context.getWrongAuthCredentials() ? null : context.getWrongAuthCredentials().toString());
+        expansionContext.put("userSelectScaType", null == scaType ? null : scaType);
```

Comment:
null == scaType ? null : scaType this expression doesn't make sense

Reply:
I deleted if case and changed it to scaType